pkg/cluster/k8s.go

```diff
@@ -41,9 +39,9 @@ func NewK8sProvisioner() (clusterTypes.Provisioner, error) {
 func (k *K8sProvisioner) SetUp(ctx context.Context, spec interface{}) ([]clusterTypes.Node, []clusterTypes.ClientNode, error) {
 	switch s := spec.(type) {
 	case *tidb.TiDBClusterRecommendation:
-		return k.setUpTiDBCluster(ctx, s)
+		return k.setUpTiDBCluster(s)
 	case *binlog.ClusterRecommendation:
-		return k.setUpBinlogCluster(ctx, s)
+		return k.setUpBinlogCluster(s)
```

Comment:
why remote context here

Reply:
ctx isn't used now, and the process `setUpBinlogCluster` may last a long time especially when lack of resources, so I think using `fixture.Context.WaitClusterReadyDuration` to control the wait time for cluster ready is better.